### DISCUSSION TITLE
Batch file not recognised.

### DISCUSSION THREAD

**Discussion by MaxsRepo** [pyinstaller/pyinstaller#8047] [Help] [answered]
Hi there, I'm new to Python and pyinstaller.

I added Python to my path, executed `pyinstaller --onefile <filename>`,  but my cmd gives the issue:

`'pyinstaller' is not recognized as an internal or external command,
operable program or batch file.`

As I said, I'm new to this, so can someone give me some help, please?

**Comment by rokm** [comment, marked as answer]
You need to add Python's Scripts directory to `PATH`. Or run PyInstaller as a module instead: `python -m PyInstaller ...`.

**Comment by milisonagithub** [comment]
python -m PyInstaller -F <python script>.py
